Lib/typing.py

```diff
@@ -1158,6 +1158,13 @@ def __copy__(self):
                               self.__parameters__, self.__args__, self.__origin__,
                               self.__extra__, self.__orig_bases__)
 
+    def __setattr__(self, attr, value):
+        # We consider all the subscripted genrics as proxies for original class
+        if attr.startswith('__') and attr.endswith('__'):
+            super(GenericMeta, self).__setattr__(attr, value)
```

Comment:
I have a small question, why not just using `super().__setattr__(attr, value)` but `super(GenericMeta, self)`? Isn't that equivalent at this case?